setup_f5_eks.sh

```diff
@@ -348,6 +364,74 @@ fi
 echo -e "\nInstalling Fusion 5.0 Helm chart ${CHART_VERSION} into namespace ${NAMESPACE} with release tag: ${RELEASE}"
 
 
+INGRESS_VALUES=""
+if [ "${INGRESS_HOSTNAME}" != "" ] && [ "${DEPLOY_ALB}" == "1" ]; then
+
+  #Creating required ALB policy and attaching it to the Cluster Node Group
+
+  wget https://raw.githubusercontent.com/kubernetes-sigs/aws-alb-ingress-controller/v1.1.5/docs/examples/iam-policy.json -O alb-policy.json
+
+  NODE_GROUP_ARN=$(aws --profile "${AWS_ACCOUNT}" --region "${REGION}" cloudformation describe-stack-resource --stack-name eksctl-${CLUSTER_NAME}-nodegroup-standard-workers --logical-resource-id NodeInstanceRole --query 'StackResourceDetail.PhysicalResourceId' --output text)
+
+  POLICY_ARN=$(aws --profile "${AWS_ACCOUNT}" --region "${REGION}" iam create-policy --policy-name eksctl-${CLUSTER_NAME}-alb-policy --policy-document file://alb-policy.json --query 'Policy.Arn' --output text)
+
+  aws --profile "${AWS_ACCOUNT}" --region "${REGION}" iam attach-role-policy --role-name ${NODE_GROUP_ARN} --policy-arn ${POLICY_ARN}
+
+
+  #Installing ALB
+
+  echo -e "\nInstalling Alb Ingress Controller"
+
+  helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com
+  helm install incubator/aws-alb-ingress-controller --set autoDiscoverAwsRegion=true --set autoDiscoverAwsVpcID=true --set clusterName=${CLUSTER_NAME} --generate-name
```

Comment:
Do we need to check if the incubator repo already exists? Do you get a weird error message if you already have the repo?

Do we need to run helm repo update after adding?

Reply:
I have added a check for incubator and the repo update command